test/methods.js

```diff
@@ -618,6 +618,55 @@ test( "remote, highlight all invalid fields", function( assert ) {
 		done();
 	}, 500 );
 } );
+test( "remote, unhighlighted should be invoked after being highlighted/invalid", function( assert ) {
+	expect( 2 );
+
+	var done1 = assert.async(),
+		done2 = assert.async(),
+		$form = $( "#testForm25" ),
+		$somethingField = $form.find( "input[name='something25']" ),
+		responseText = "false",
+		response = function() { return responseText; },
+		validateOptions = {
+			highlight: function( e ) {
+				$( e ).addClass( "error" );
+			},
+			unhighlight: function( e ) {
+				$( e ).removeClass( "error" );
+			},
+	        rules: {
+				something25: {
+	                required: true,
+	                remote: {
+	                    url: "callback.php",
+	                    type: "post",
+						data: {
+							responseText: response
+						},
+						async: false
```

Comment:
The use of `synchronous` calls is discouraged due to the negative effects to the user experience.